style.css

```diff
@@ -1,18 +1,652 @@
-/*@import "fonts.css";*/
-
-/*body {
-	font-family: "Montserrat-SemiBold";
-}*/
-
 .header, .start, .specifications, .salon, .description__title, .parameters__item, .safety__title, .stars__subtitle, .fivestars__textstars{
-	font-family: "Montserrat-SemiBold";
+font-family: "Montserrat-SemiBold";
 }
 .specifications__item, .description__link, .stars__item, .fivestars__text, .fivestars__link{
-	font-family: "Montserrat-Bold";
+font-family: "Montserrat-Bold";
 }
 .description__text, .safety__text{
-	font-family: "Montserrat-Medium";
+font-family: "Montserrat-Medium";
 }
 .parameters__small{
-	font-family: "Montserrat-Regular";
+font-family: "Montserrat-Regular";
+}
+
+
+
+
+
+
+
+
+
+body{
+max-width: 1440px;
+border: 1px solid orange;
+margin: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html_rus3/pullRequest?prid=6047524)